Changes

```diff
@@ -1,6 +1,36 @@
 Release history of Zonemaster
 
 
+v2024.2.1 2025-03-04
+
+ [Release information]
+ - Translations have not been fully updated in this release. They will
+   be updated in an upcoming extra release.
+
+ [Features]
+- Adds Slovenian translation (#1333). Also see Zonemaster-Engine, Zonemaster-CLI,
+  Zonemaster-Backend and Zonemaster-GUI.
+
+ [Fixes]
+- Fixes typo in documentation (#1347)
```

Comment:
```suggestion
- Documentation fixes (#1338, #1347)
```

Reply:
I interpreted #1338 to be something equivalent of CI changes.